end-to-end-tests/pageObjects/pageEditor/modListingPanel.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2024 PixieBrix, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { BasePageObject } from "../basePageObject";
+import { uuidv4 } from "@/types/helpers";
+import { type Locator } from "@playwright/test";
+
+// Starter brick names as shown in the Page Editor UI
+export type StarterBrickName =
```

Comment:
To get rid of comment?
```suggestion
export type StarterBrickUIName =
```